CYCLONEDDS_QUALITY_DECLARATION.md

```diff
@@ -14,65 +14,90 @@ Below are the rationales, notes, and caveats for this claim, organized by each r
 
 ### Version Scheme [1.i]
 
-There is a three-part version attached to `CycloneDDS` releases.
-The sources define the version parts as `MAJOR`.`MINOR`.`PATCH`.
-The CMake sources configure the compatibility mode to [`SameMajorVersion`](https://cmake.org/cmake/help/latest/module/CMakePackageConfigHelpers.html#generating-a-package-version-file), indicating that breaking changes are accompanied by a change to the `MAJOR` version part.
-There is no explicit policy regarding versioning in the documentation for `CycloneDDS`, however there is some information in the [Releases section](https://www.eclipse.org/projects/handbook/#release) of the Eclipse project handbook which discusses _Major_, _Minor_, and _Service_ release criteria.
-It is possible that these release categories align align with changes to the three version parts, but this is not explicitly stated in either `CycloneDDS` documentation or in the Eclipse project handbook.
+`CycloneDDS` versioning follows the standard semantic versioning except that major version 0 is also considered stable. `CycloneDDS` was already a stable code base when it was contributed to [The Eclipse Foundation](https://eclipse.org)
+
+* MAJOR version when you make incompatible API changes.
```

Comment:
I don't think this makes sense as generic second person. The reader of this document is generally trying to understand how the package's maintainers control versioning, not how to version the package itself. I'd suggest something along the lines of:

i.e., "The MAJOR version shall be incremented when an incompatible API change is made"

Reply:
ok, thanks!